src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/DocumentSpanEntry.cs

```diff
@@ -201,6 +201,25 @@ private static Span GetRegionSpanForReference(SourceText sourceText, TextSpan so
                     sourceText.Lines[firstLineNumber].Start,
                     sourceText.Lines[lastLineNumber].End);
             }
+
+            bool ISupportsNavigation.TryNavigateTo(bool isPreview)
+            {
+                // If the document is a source generated document, we need to do the navigation ourselves;
+                // this is because the file path given to the table control isn't a real file path to a file
+                // on disk.
```

Comment:
interesting.  what file path do we give it?  what does it show the user?

Reply:
also, why not just do this for 100% of all results?  seems like it would work nicely that way.

Reply:
> what does it show the user?

We just show a relative path which is the assembly name/generator name path. For example the highlighted line here:

![image](https://user-images.githubusercontent.com/201340/104768347-5581b300-5722-11eb-860b-6f27be985391.png)

> also, why not just do this for 100% of all results?

Admittedly risk reduction. Looking through some of the editor code I think they also had some smarts around if you have a file path and then edits happen, it might be able to update the line/column numbers; that won't be useful in our case here.